kedro/framework/cli/catalog.py

```diff
@@ -13,12 +13,7 @@
 
 def _create_session(package_name: str, **kwargs):
     kwargs.setdefault("save_on_close", False)
-    try:
-        return KedroSession.create(package_name, **kwargs)
-    except Exception as exc:
-        raise KedroCliError(
-            f"Unable to instantiate Kedro session.\nError: {exc}"
-        ) from exc
+    return KedroSession.create(package_name, **kwargs)
```

Comment:
Are the changes in this file related to the logging thing?

Reply:
Yes - sorry I plan to drop a few comments here but I never got to do it. Basically the error was there only because when `session` is initialised, it read `config["logging"]`. This is now delayed until `catalog` is read because `KedroSession.create` doesn't read config anymore.